cmd/buildah/images.go

```diff
@@ -373,6 +374,13 @@ func formattedSize(size int64) string {
 }
 
 func outputUsingTemplate(format string, params imageOutputParams) error {
+	matched, err := regexp.MatchString("{{.*}}", format)
+	if err != nil {
```

Comment:
Nit: can you combine these two lines together
```
if matched, err := regexp.MatchString("{{.*}}", format); err != nil {
...
```

Reply:
Fine, updated.